orc/src/main/java/org/apache/iceberg/orc/OrcFileAppender.java

```diff
@@ -147,11 +146,12 @@ public long length() {
   @Override
   public List<Long> splitOffsets() {
     Preconditions.checkState(isClosed, "File is not yet closed");
-    try (Reader reader = ORC.newFileReader(file.toInputFile(), conf)) {
-      List<StripeInformation> stripes = reader.getStripes();
+    try {
+      List<StripeInformation> stripes = writer.getStripes();
       return Collections.unmodifiableList(Lists.transform(stripes, StripeInformation::getOffset));
     } catch (IOException e) {
-      throw new RuntimeIOException(e, "Can't close ORC reader %s", file.location());
+      throw new RuntimeIOException(
+          e, "Cannot receive stripe information from writer for %s", file.location());
```

Comment:
"Cannot receive" seems like a strange way to state that `getStripes` failed. What about `Failed to get stripe information from writer for: %s`?

Reply:
Makes sense, will change.